docs/sources/configure-client/language-sdks/nodejs.md

```diff
@@ -49,7 +49,7 @@ Pyroscope.start()
 
 [comment]: <> (TODO This needs its own page like https://grafana.com/docs/pyroscope/latest/configure-client/grafana-agent/go_pull/)
 {{< admonition type="note" >}}
-If you'd prefer, you can use Pull mode using [Grafana Alloy](https://grafana.com/docs/alloy/latest/) (recommended) or [Grafana Agent]({{< relref "../grafana-agent" >}}) (legacy).
+If you'd prefer, you can use Pull mode using [Grafana Alloy](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/configure-client/grafana-agent/).
```

Comment:
should we link to the pull mod page directly ? https://grafana.com/docs/pyroscope/latest/configure-client/grafana-agent/go_pull/

Reply:
DOne.